### ISSUE TITLE
Not parsing all fields from PostgreSQL DDL

### ISSUE THREAD

**Issue by mudlee** [shinichi-takii/ddlparse#24]
I use 1.2.0, this way:
```python
DdlParse().parse(ddl=sql_ddl, source_database=DdlParse.DATABASE.postgresql).to_bigquery_fields()
```

## Input:

```SQL
CREATE TABLE my_table (
    id integer NOT NULL,
    user_id integer,
    first_name character varying,
    last_name character varying,
    birth_date timestamp without time zone,
    gender_status integer,
    marital_status integer,
    street_address text,
    zip_code character varying,
    city character varying,
    country character varying,
    handphone character varying,
    created_at timestamp without time zone NOT NULL,
    updated_at timestamp without time zone NOT NULL,
    pending_deposit_amount numeric(18,4) DEFAULT 0.0,
    free_amount numeric(18,4) DEFAULT 0.0,
    secured_pledge_amount numeric(18,4) DEFAULT 0.0,
    unsecured_pledge_amount numeric(18,4) DEFAULT 0.0,
    pending_withdraw_amount numeric(18,4) DEFAULT 0.0
);
```

## Output:
```json
[
  {
    "name": "id",
     "type": "INTEGER",
     "mode": "REQUIRED"
  },
  {
    "name": "user_id",
     "type": "INTEGER",
     "mode": "NULLABLE"
  },
  {
    "name": "first_name",
     "type": "STRING",
     "mode": "NULLABLE"
  },
  {
    "name": "last_name",
     "type": "STRING",
     "mode": "NULLABLE"
  },
  {
    "name": "birth_date",
     "type": "DATETIME",
     "mode": "NULLABLE"
  },
  {
    "name": "gender_status",
     "type": "INTEGER",
     "mode": "NULLABLE"
  },
  {
    "name": "marital_status",
     "type": "INTEGER",
     "mode": "NULLABLE"
  },
  {
    "name": "street_address",
     "type": "STRING",
     "mode": "NULLABLE"
  },
  {
    "name": "zip_code",
     "type": "STRING",
     "mode": "NULLABLE"
  },
  {
    "name": "city",
     "type": "STRING",
     "mode": "NULLABLE"
  },
  {
    "name": "country",
     "type": "STRING",
     "mode": "NULLABLE"
  },
  {
    "name": "handphone",
     "type": "STRING",
     "mode": "NULLABLE"
  },
  {
    "name": "created_at",
     "type": "DATETIME",
     "mode": "REQUIRED"
  },
  {
    "name": "updated_at",
     "type": "DATETIME",
     "mode": "REQUIRED"
  },
  {
    "name": "pending_deposit_amount",
     "type": "FLOAT",
     "mode": "NULLABLE"
  }
]
```

**Comment by mudlee** [comment]
Based on my tests, `DEFAULT 0.0` causes the problem

**Comment by shinichi-takii** [comment]
@mudlee 
Thank you for the report.
I fixed issue to v1.2.1.
